templates/registration/includes/profile_music.html

```diff
@@ -1,12 +1,34 @@
 <div class="investments">
-    <h2>MY MUSIC</h2>
+    <h2>MUSIC</h2>
     <div class="break">
         <hr>
     </div>
     <div class="row small-up-1 medium-up-2 large-up-4 centered artistlist">
         {% for album in albums %}
             <div class="column">
-                {% include "object_preview.html" with object=album img=album.artwork.img %}
+                <div class="artist-list-image img-list">
+                    {% include "object_preview.html" with object=album img=album.artwork.img %}
+                </div>
+                <div>
+                    <p>
+                    {% for disc in album.discs %}
+                        {% for track in disc %}
+                            {% if track.total_streams %}
+                                <i class="fa fa-cloud" aria-hidden="true"></i>
+                                {{ track.total_streams}}
+                            {% else %}
+                                EXCLUSIVE
```

Comment:
Yeah, right now when an album or song isn't available in stores it shows 0 streams 0 downloads. This is a bit confusing and doesn't look clean. If a song isn't released yet I want this part to say that it's an exclusive. 

Wasn't exactly sure the best way to do that so I just made it say exclusive if there are no streams. Would be great if there is a better way to do this.